index.html

```diff
@@ -1,19 +1,91 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+  <body>
+    <header class="header">
+      <img src="img/karma-logo.svg" alt="Karma Logo" class="nav-logo" />
 
-</body>
+      <nav>
+        <ul class="nav-list">
+          <li class="nav-item-meet-karma">
+            <a class="nav-link-meet-karma" href="#">Meet Karma</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">How it Works</a>
+          </li>
+          <li class="nav-item">
```

Comment:
It looks we repeat nav-item and nav-link styles a lot. Let's drop this code duplication from HTML and handle the styling in CSS

Reply:
I don't think I understand what you mean. I had already styled in CSS not HTML. These are class names. Do you want me to remove class names(nav-item, nav-link) ?

Reply:
Currently you have:

```html
<ul class="nav-list">
  <li class="nav-item">
    <a class="nav-link" href="#">Blog</a>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="#">How it Works</a>
  </li>
</ul>
```

What I'm suggesting is:

```html
<ul>
  <li>
    <a href="#">Blog</a>
  </li>
  <li>
    <a href="#">How it Works</a>
  </li>
</ul>
```

You can still apply the styles you want in CSS. But this makes HTML more compact and readable. 

Usually, when you repeat the same thing over and over (styles for each navigation item in our case) - it's a good sign there is a way to simplify and reduce the code duplication.